shell/client/grain-client.js

```diff
@@ -39,6 +39,18 @@ const promptNewTitle = function () {
   }
 };
 
+const showWhoHasAccess = function () {
+  const closer = globalTopbar.addItem({
+    name: "who-has-access",
+    template: Template.whoHasAccess,
+    popupTemplate: Template.whoHasAccessPopup,
+    startOpen: true,
+    onDismiss: function () {
+      return "remove";
+    }
```

Comment:
The stylechecks want the trailing comma here, so you can add/remove elements to/from the object without needing to change unrelated lines.